ocs_ci/ocs/machine.py

```diff
@@ -191,7 +195,8 @@ def create_custom_machineset(
     Args:
         role (str): Role type to be added for node eg: it will be app,worker
         instance_type (str): Type of aws instance
-        label (str): Label to be added to the node
+        label (list of tuples): List of Labels (key, val) to be added to the node
+        taints (list of dict): List of taints to be applied
```

Comment:
```suggestion
        label (list): List of Labels (key, val) to be added to the node
        taints (list): List of taints to be applied
```
The types we specify here need to be actual python types. If we want to elaborate on what kind of information is in the list we can do that in the description.

Reply:
ack

Reply:
done